src/schema/clan/invite.ts

```diff
@@ -0,0 +1,23 @@
+import { Type } from "@sinclair/typebox";
+
+import { defineEndpoint } from "@/generator-helpers.js";
+
+export default defineEndpoint({
+    source: "user",
+    target: "server",
+    description: "Invite the target player to your clan.",
+    request: {
+        data: Type.Object({
+            userId: Type.Ref("userId"),
+            clanId: Type.Ref("clanId"),
+        }),
+    },
+    response: [
+        { status: "success" },
+        { status: "failed", reason: "clan_not_available" },
+        { status: "failed", reason: "user_not_available" },
+        { status: "failed", reason: "no_permission" },
+        { status: "failed", reason: "already_invited" },
+        { status: "failed", reason: "already_member" },
```

Comment:
we already use `invalid_user` in a few places, which should be preferred over `user_not_available`. Similarly `invalid_clan` is more consistent.

I think `already_invited` should be dropped and the invite should succeed if there is already a pending invite. Because in this case, it's not like the user can do anything upon seeing the error, and it effectively means the action was carried out.